games-arcade/rocksndiamonds/additional-files/rocksndiamonds.rdef.in

```diff
@@ -0,0 +1,38 @@
+// Rocks 'n' Diamonds icon by Stephanie Fu
+
+resource app_signature "application/x-vnd.Artsoft-rocksndiamonds";
+
+resource app_flags B_EXCLUSIVE_LAUNCH;
+
+resource app_version {
+    major  = @MAJOR@,
+    middle = @MIDDLE@,
+    minor  = @MINOR@,
+    variety = B_APPV_DEVELOPMENT,
+    internal = @INTERNAL@,
+    short_info = "Rocks'n'Diamonds",
+    long_info = "Rocks'n'Diamonds, an arcade style game.\nCopyright 2001-2013 Artsoft Entertainment"
+};
+
+resource file_types message;
+
+resource vector_icon array {
+    $"6E6369660B03C6630203904902020004025AD37E28FFAB6115030065FE03A9CB"
+    $"FE033287FE05FE0300307E05FF04016303010000100A04243E3A483A5E24540A"
+    $"043A48503E50543A5E0A04243E3A48503E3A340A043C404440473739370A0339"
+    $"37402647370A0439373432373B3C400A033432402639370A0347374C3444400A"
+    $"03402647374C340607FE1B312EBA85B854BA1CB8A4BAA5B83BBAC5B7A8BACAB7"
+    $"E5BABFB77033B72233B75233B6D4BA1FB6BCBA5FB6E7B999B6BC30292D060C9A"
+    $"BFAA3035B86DBAAAB57C31273428B9ED28BB772935B6D533B66FBBA0B6F7BB84"
+    $"B83ABBB7B7FABB2AB8ACBAC5B929BBF0BCE933BC3D3231302F0608AFF9382FBC"
+    $"F0B7D737323532BC4DB9DC323333332F3526322535243225BBBEB4773927BC43"
+    $"B51F39B6FD06096BEE0335273628BACDB5393328B9E9B660B9C43530362FBA32"
+    $"2F36B743BBF2B80E362A382A362A362A362A38290608F22F3A59474C57C328C9"
+    $"89C4A6C8DC5356C6E1C8B7535655545554C840C7A85553C7FCC76155535D4D50"
+    $"0A06243B24513A5B5051503B3A310A062E313A2546333E3F363F313A0E0A0901"
+    $"0D2020220A00010020201D0A01010120201D0A02010220201D0A0A010E100117"
+    $"8200040A0A010F1001178400040A030103201A1F0A040104201A1F0A05010520"
+    $"1A1F0A060106201A1F0A070107201A1F0A050108201A1F0A0802090A201D3C0A"
+    $"08020B0C20323F"
```

Comment:
Is that file about 1682 chars long?
If you type
```
cat rocksndiamonds.rdef.clean
```
do you see what you expect?
If you rename it to rocksndiamonds.rdef.txt, can you open it?